src/Illuminate/Database/Query/Grammars/PostgresGrammar.php

```diff
@@ -3,6 +3,7 @@
 namespace Illuminate\Database\Query\Grammars;
 
 use Illuminate\Database\Query\Builder;
+use Illuminate\Support\Str;
```

Comment:
Within this project, imports should be sorted by line-length, and then alphabetically.